components/molecule/phoneInput/test/index.test.js

```diff
@@ -255,4 +256,27 @@ describe('MoleculePhoneInput', () => {
 
     expect(input).to.not.be.null
   })
+
+  it('Should apply different mask given a land line value', () => {
```

Comment:
👏🏻 Thanks for the specific test.

Reply:
This component is like my baby, and my baby has a 100% coverage 💘